src/SourceBuild/content/eng/pipelines/templates/stages/vmr-scan.yml

```diff
@@ -36,6 +36,7 @@ stages:
         ./.dotnet/dotnet darc vmr scan-binary-files
         --vmr "$(Build.SourcesDirectory)"
         --tmp "$(Agent.TempDirectory)"
+        --baseline-file "(Build.SourcesDirectory)\src\installer\src\VirtualMonoRepo\baseline-file.txt"
```

Comment:
We're running on Linux so forward slashes

Reply:
fixed